test/examples/src/index.md

```diff
@@ -199,7 +199,7 @@ julia> g(2)
 ```
 ```jldoctest
 julia> f(2)
-ERROR: UndefVarError: f not defined
+ERROR: UndefVarError: `f` not defined in `Main`
```

Comment:
This version of the message is shown in Julia >= 1.11; this is one of the reasons why we have `warnonly = (VERSION.major == 1 && VERSION.minor <= 10)` further above